### ISSUE TITLE
Warning: Attempt to present <RNSpotifyAuthController: 0x7ff563a58e00> on <RCTModalHostViewController: 0x7ff55dd45cf0> whose view is not in the window hierarchy!

### ISSUE THREAD

**Issue by sslash** [lufinkey/react-native-spotify#145]
*Platform*: ios
*React-native version:*: 0.61.2
*rn-spotify-sdk*: 1.2.11

This error suddenly started happening after I placed the Spotify authentication screen in a different place in my app:

```
Warning: Attempt to present <RNSpotifyAuthController: 0x7fc60a4b5200> on <RCTModalHostViewController: 0x7fc609040960> whose view is not in the window hierarchy!
```

The error happens after a call to `Spotify.initialize` followed by `Spotify.login`

Any idea what could trigger this?

**Comment by lufinkey** [comment]
To be honest no idea. I present the auth controller on whatever the top visible view controller is in the app

**Comment by lufinkey** [comment]
@sslash did you ever find anything on this?

**Comment by sslash** [comment]
Not really. I've just settled with the fact that we have to make sure that any other modal must be completely removed before we show the Spotify authentication modal

**Comment by lufinkey** [comment]
Any chance you could post a view controller stack for when it fails?

**Comment by sslash** [comment]
I'll try, yeah!
